cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require( '../polyfill/class-list' );
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - DOM element to get siblings of.
+ * @param {string} className - CSS classname.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/c07d478f0aaccd5e088a19cae7003fbfed3d7708